pkg/e2e/operators/cloudingress/deletetapps2.go

```diff
@@ -0,0 +1,133 @@
+package cloudingress
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
+	"github.com/openshift/osde2e/pkg/common/constants"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/e2e/operators"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
+	h := helper.New()
+	ginkgo.Context("Delete apps2 ingresscontroller", func() {
+		ginkgo.It("Should not be able to delete the ApplicationIngress that doesn't belong to CIO", func() {
+			//1. create a secondaryIngress that does belong to cloud-ingress-operator
+			secondaryIngress := secondaryIngress(h)
+
+			//only create the secondary ingress if it doesn't exist already in the publishing strategy
+			if _, exists, _ := appIngressExits(h, false, secondaryIngress.DNSName); !exists {
+				addAppIngress(h, secondaryIngress)
+			}
+			time.Sleep(time.Duration(30) * time.Second)
+			log.Print("Created a secondary ingress")
+			// from DNSName app-e2e-apps.cluster.mfvz.s1.devshift.org,
+			// the ingresscontroller name is app-e2e-apps: everything before the first period
+			ingressControllerName := strings.Split(secondaryIngress.DNSName, ".")[0]
+			// check that the ingresscontroller app-e2e-apps was created
+			ingressControllerExists(h, ingressControllerName, true)
+			log.Printf("Check to see of the ingressControllerExists")
+			// check if the secondary router is created
+			// the created router name should be router-app-e2e-apps
+			deploymentName := "router-" + ingressControllerName
+			deployment, err := operators.PollDeployment(h, "openshift-ingress", deploymentName)
+			log.Print("Check to see if the secondary router exists")
+			Expect(err).ToNot(HaveOccurred(), "failed fetching deployment")
+			Expect(deployment).NotTo(BeNil(), "deployment is nil")
+
+			// wait 1 minute for all routers to start
+
+			time.Sleep(time.Duration(60) * time.Second)
+			//2.delete the annotation
+			apps2Ingress, _ := getingressController(h, ingressControllerName)
+			log.Printf("The ingresscontroller object annotation currently is: %+v\n", apps2Ingress.ObjectMeta.Annotations)
```

Comment:
This seems like a lot of text to print out. Are you using these logs as debug statements? Are they needed during the e2e run itself?